docs/UserGuide.md

```diff
@@ -162,9 +163,8 @@ You can also mark the specified elderly as called on a specific date and add not
 Format: `mark INDEX/NRIC [d/DATE] [o/NOTES]`
 
 * Marks the elderly at the specified `INDEX` or `NRIC`.
-* The `INDEX` refers to the number displayed in the current person list. It must be a **positive integer** (e.g., 1, 2, 3, …​) and fall within the range of the list shown.
-* The `NRIC` must be a valid, government issued NRIC.
-* The `NRIC` used to select the elderly to mark must exist in the current person list.
+* The `INDEX` refers to the index number shown in the displayed person list. It must be a **positive integer** (e.g., 1, 2, 3, …​) and fall within the range of the list shown.
+* The `NRIC` used to select the elderly to mark must be a valid, government issued NRIC and must exist in the current person list.
```

Comment:
Could use `displayed` instead of `current` to maintain consistency. Also in line 186. Up to you.